projects/dashboards/src/widgets/widget-renderer.ts

```diff
@@ -60,7 +60,7 @@ export abstract class WidgetRenderer<TModel extends object, TData = unknown> imp
 
   private fetchAndRunChangeDetection(): void {
     this.data$ = this.fetchData().pipe(
-      shareReplay(),
+      shareReplay({ bufferSize: 1, refCount: true }),
```

Comment:
not needed, since fetchData is not a long-lived stream

Reply:
(that said, moving the shareReplay here to the bottom of the pipe would be appropriate)